aldo changes

AldoRamirez97 opened the pull request.

AldoRamirez97 committed:
aldo changes

AldoRamirez97 committed:
new changes

ryanc36 commented on Animal.py:
```diff
@@ -1,7 +1,8 @@
 class Animal:
-    def __init__(self, name, walk):
+    def __init__(self, name, walk, eat):
```
We can keep this to add additional parameters/arguments to the Animal class

ryanc36 commented on Animal.py:
```diff
@@ -1,7 +1,8 @@
+    def __init__(self, name, walk, eat):
         self.__name = name 
         self.__walk = walk
+        self.__eat = eat
```
eat

ryanc36 commented on Animal.py:
```diff
@@ -10,6 +11,6 @@ def talk(self):
 
-    #def eat(self):
-        #print(f"{self.__name} is eating.")
+    def eat(self):
```
Looks good

ryanc36 committed:
Delete Usage.ipynb

Deleted because it was displaying juypter notebook instead of python code and was not needed
